addons/microsoft_calendar/models/calendar_recurrence_rule.py

```diff
@@ -19,8 +16,19 @@ class RecurrenceRule(models.Model):
 
     microsoft_id = fields.Char('Microsoft Calendar Recurrence Id')
 
+    def _compute_rrule(self):
+        for recurrence in self:
+            if recurrence.rrule != recurrence._rrule_serialize():
+                recurrence.write({'rrule': recurrence._rrule_serialize()})
+
+    def _inverse_rrule(self):
+        for recurrence in self.filtered('rrule'):
+            values = self._rrule_parse(recurrence.rrule, recurrence.dtstart)
+            # We avoid to trigger patch in the inverse
+            recurrence.write(values)
+
```

Comment:
I think you can remove this inverse, it does nothing different than the method in calendar_recurrence.py file